package.json

```diff
@@ -1182,8 +1182,8 @@
     "@babel/parser": "^7.24.6",
     "@babel/traverse": "^7.24.6",
     "@mongodb-js/compass-components": "^1.25.0",
```

Comment:
they should actually already be using the latest version because we use the caret range https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004
but let me make it official and bump to 1.30.0